.pre-commit-config.yaml

```diff
@@ -5,24 +5,7 @@ repos:
       - id: pretty-format-json
         name: Pretty Format JSON
         args: [--autofix, --no-sort-keys]
-  - repo: https://github.com/pre-commit/mirrors-eslint
```

Comment:
this is just done in the ci run to bypass issues with pre-commit not playing nice with eslint 9